Dockerfile

```diff
@@ -11,20 +11,24 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 
 # set default build arguments
-ARG SDK_VERSION=commandlinetools-linux-7302050_latest.zip
+ARG SDK_VERSION=commandlinetools-linux-9123335_latest.zip
+ARG ANDROID_BUILD_VERSION=31
+ARG ANDROID_TOOLS_VERSION=30.0.3
+ARG BUCK_VERSION=2022.05.05.01
 ARG NDK_VERSION=21.4.7075529
-ARG NODE_VERSION=12.x
-ARG CMAKE_VERSION=3.6.4111459
+ARG NODE_VERSION=16
```

Comment:
we could potentially downgrade this to 14 but figured that we may as well use 16 like upstream does. >=14 is required because the smokescreen test attempts to build the latest react native, which now requires a higher node version than 12